mcrouter/lib/network/gen/MemcacheRouterStats.h

```diff
@@ -25,33 +25,37 @@ namespace facebook {
 namespace memcache {
 
 struct MemcacheRouterStatsConfig {
-  static constexpr size_t kNumRequestGroups = 6;
+  static constexpr size_t kNumRequestGroups = 7;
   static constexpr std::array<folly::StringPiece, 1 * kNumRequestGroups>
       sumStatNames{{folly::StringPiece("cmd_delete_count"),
                     folly::StringPiece("cmd_get_count"),
                     folly::StringPiece("cmd_lease_get_count"),
                     folly::StringPiece("cmd_lease_set_count"),
                     folly::StringPiece("cmd_set_count"),
-                    folly::StringPiece("cmd_other_count")}};
+                    folly::StringPiece("cmd_other_count"),
+                    folly::StringPiece("cmd_gets_count")}};
```

Comment:
This list is order dependent. The index of each of these sum metrics is returned below.